lib/basedriver/commands/log.js

```diff
@@ -0,0 +1,39 @@
+import log from '../logger';
+import _ from 'lodash';
+
+
+const commands = {}, helpers = {}, extensions = {};
+
+// override in sub-classes, with appropriate logs
+// in the form of
+//   {
+//     type: {
+//       description: 'some useful text',
+//       getter: () => {}, // some function that will be called to get the logs
+//     }
+//   }
+extensions.supportedLogTypes = [];
```

Comment:
I assume it has be an object rather that an array

Reply:
Ah, yes. Thanks! This works for an empty array, since it is just an object same as `{}`.